src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -198,19 +201,10 @@ public override async Task<CodeAction> GetFixAsync(FixAllContext fixAllContext)
             ConcurrentBag<(Diagnostic diagnostic, CodeAction action)> fixes,
             FixAllState fixAllState, CancellationToken cancellationToken)
         {
-            Debug.Assert(!diagnostics.IsDefault);
-            cancellationToken.ThrowIfCancellationRequested();
-
-            var registerCodeFix = GetRegisterCodeFixAction(fixAllState, fixes);
-            var context = new CodeFixContext(
-                project, diagnostics, registerCodeFix, cancellationToken);
-
-            // TODO: Wrap call to ComputeFixesAsync() below in IExtensionManager.PerformFunctionAsync() so that
-            // a buggy extension that throws can't bring down the host?
-            return fixAllState.CodeFixProvider.RegisterCodeFixesAsync(context) ?? Task.CompletedTask;
```

Comment:
📝 I'm hoping this wasn't used. Normal code fixes don't support no-location diagnostics, and the SuppressMessage batch fixer overrides this method anyway.

Reply:
Yes, I think this is only used by suppression fixer when invoked for bulk suppressing diagnostics from error list, which can include project diagnostics.